src/test/java/ui/tests/AuthTest.java

```diff
@@ -0,0 +1,193 @@
+        logger.info("Email generation");
+        String email = "flynn+" + RandomStringUtils.randomAlphanumeric(10) + "@gmail.com";
+        logger.info("Password generation");
+        String password = RandomStringUtils.randomAlphanumeric(10);
+        logger.info("Postal code generation");
+        String postCode = RandomStringUtils.randomNumeric(5);
+        logger.info("Mobile phone generation");
+        String mobilePhone = "+" + RandomStringUtils.randomNumeric(10);
+        logger.info("Opening main page and clicking Sign in button");
+
+        mainPage.clickSignInButton();
+        logger.info("Starting account registration with generated email");
+
+        authenticationPage.startAccountAuth(email);
+
+        logger.info("Read User Object to Json");
+        UserAccount user = new UserAccount("Johnny", "Flynn", password, email, "14", "3", "1983",
+                "South Africa", "Johannesburg",
+                "10", postCode, mobilePhone, "Home");
+        userAccountsPool.serialize("src/test/resources/accountData.json", user);
+
+        logger.info("Create expected account");
+        UserAccountResult expectedAccount =
+                new UserAccountResult("Johnny", "Flynn", email, "14", "3", "1983");
+
+        logger.info("Read Json to User Object");
+        userAccountsPool.deserialize("src/test/resources/accountData.json");
+
+        logger.info("Get User from User Account Pool");
+        UserAccount userPool = userAccountsPool
+                .getUser("src/test/resources/accountData.json");
+
+        logger.info("Account registration");
+        accountCreationPage.createAccount(userAccountsPool
+                .getUser("src/test/resources/accountData.json"));
+        logger.info("Signing out");
+        accountPage.signOut();
+        logger.info("Opening main page and clicking Sign in button");
+        mainPage.clickSignInButton();
+
+        logger.info("Sign In");
+        authenticationPage.logIn(user.getEmail(), user.getPassword());
+
+        logger.info("Open Personal Info page");
+        accountPage.openPersonalInfo();
+
+        logger.info("Create actual account");
+        UserAccountResult actualAccount =
+                new UserAccountResult(driver.findElement(By.name("firstname")).getAttribute("value"),
+                        driver.findElement(By.name("lastname")).getAttribute("value"),
+                        driver.findElement(By.name("email")).getAttribute("value"),
+                        new Select(driver.findElement(By.id("days"))).getFirstSelectedOption().getAttribute("value").trim(),
+                        new Select(driver.findElement(By.id("months"))).getFirstSelectedOption().getAttribute("value").trim(),
+                        new Select(driver.findElement(By.id("years"))).getFirstSelectedOption().getAttribute("value").trim());
+
+        logger.info("Verify created account");
+        Assert.assertEquals(expectedAccount, actualAccount);
```

Comment:
If you use `assertEquals` please override `toString` as well, so that assert message will be more readable